src/main/java/pl/javastart/task/ChristmasTreeDescription.java

```diff
@@ -0,0 +1,13 @@
+package pl.javastart.task;
+
+public class ChristmasTreeDescription {
+    String treeColour;
+    double treeHight;
```

Comment:
Skoro jesteśmy w klasie ChristmasTree, to nie ma potrzeby dodawać "tree" do każdego pola. Wystarczy color i height (przy hight masz literówkę)